src/app/home/tabs/search/search.component.ts

```diff
@@ -0,0 +1,36 @@
+import { CommonModule } from '@angular/common';
+import { Component, inject } from '@angular/core';
+import {
+  IonInput,
+  IonItem,
+  IonCard,
+  IonCardHeader,
+  IonCardTitle,
+  IonCardContent,
+  IonButton,
+} from '@ionic/angular/standalone';
+import { SearchStateService } from './data-access/search-state.service';
+
+@Component({
+  selector: 'app-search',
+  templateUrl: './search.component.html',
+  styleUrls: ['./search.component.scss'],
+  standalone: true,
+  imports: [
```

Comment:
Isn't there a way to globally import Ionic components on the app.module or home.module like we do with custom angular components? Or maybe this is the new Angular way?

Reply:
Standalone Components are preferred since they are more performant: https://stackoverflow.com/questions/74558393/best-practices-standalone-components-vs-modules-in-angular-14